### PR TITLE
Chore: bump concurrently dependency for toolkit (and tree-kill security fix)

### PR DESCRIPTION
**Pull request by ryantxu**
Alternative to #27409

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -23533,6 +23533,13 @@ rxjs@6.6.2:
   dependencies:
     tslib "^1.9.0"
 
+rxjs@^6.5.2:
+  version "6.6.3"
+  resolved "https://registry.yarnpkg.com/rxjs/-/rxjs-6.6.3.tgz#8ca84635c4daa900c0d3967a6ee7ac60271ee552"
+  integrity sha512-trsQc+xYYXZ3urjOiJOuCOa5N3jAZ3eiSpQB5hIT8zGlL2QfnHLJ2r7GMkBGuIausdJN1OneaI6gQlsqNHHmZQ==
+  dependencies:
+    tslib "^1.9.0"
+
```
Comment (hugohaggmark) [inline yarn.lock:23542]: Hmm, why is RxJs being updated?

**Review by hugohaggmark — COMMENTED** [review]
Looks good but RxJs is getting bumped?

File: yarn.lock (hunk 1 above)
Comment (ryantxu) [reply yarn.lock:23542]: hymmm -- maybe related to:
https://github.com/grafana/grafana/issues/26748

somhow the yarn.lock versions are not respected

Comment (hugohaggmark) [reply yarn.lock:23542]: Maybe something else refers to RxJs?

**Comment by peterholmberg** [comment]
How is this related to the issue you linked?

**Comment by ryantxu** [comment]
We depend on tree-kill because of concurrently.  Updating concurrently includes a viraion that was tested against the updated tree-kill

**Comment by peterholmberg** [comment]
Do you mean this one https://github.com/grafana/grafana/pull/27405 ?

**Pull request closed without merging**